test/e2e/scenario/recorder/viewports.scenario.ts

```diff
@@ -18,10 +18,6 @@ const VIEWPORT_META_TAGS = `
 `
 
 test.describe('recorder', () => {
-  test.beforeEach(({ browserName }, testInfo) => {
-    testInfo.skip(browserName !== 'chromium', 'only chromium supports touch gestures emulation for now (via CDP)')
-  })
```

Comment:
skipping in `beforeEach` prevent `createTest()` to run, which is nice but then the tags used for CI optimisation fingerprinting are not set.